### PR TITLE
Readme updates for Nanoleaf and Govee

### PR DESCRIPTION
**Pull request by stefan-hoehn**
Update information about Govee and Nanoleaf in Readme

### CODE REVIEW

File: bundles/org.openhab.binding.govee/README.md [hunk 1]
```diff
@@ -137,34 +137,43 @@ Thing channel setup:
...
+        - id: color-temperature
+        channelTypeUID: system:color-temperature
+        label: Color Temperature
+        description: Controls the color temperature of the light from 0 (cold) to 100 (warm)
+        configuration: {}
+        - id: color-temperature-abs
+        channelTypeUID: govee:color-temperature-abs
+        label: Absolute Color Temperature
+        description: Controls the color temperature of the light in Kelvin
+        configuration: {}
```
Comment (lsiepel) [inline bundles/org.openhab.binding.govee/README.md:165]: What was wrong with this indentation?

**Review by lsiepel — CHANGES_REQUESTED** [review]
Can you revert the indentation on both files as they seem fine how they where before.

File: bundles/org.openhab.binding.govee/README.md (hunk 1 above)
Comment (stefan-hoehn) [reply bundles/org.openhab.binding.govee/README.md:165]: Honestly, it wasn't intended. I had issues with the PR and had to "store" the file content in a different editor and it seems that it added these tabs.
Good you caught it.

**Comment by stefan-hoehn** [comment]
This was really annoying - it took me several commits to finally fix this :-(
Good that we have squash...
Now it should be okay.

**Review by lsiepel — APPROVED** [review]
Thanks, LGTM

**Pull request merged**